src/components/Upload.js

```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react"
+
+import FineUploaderS3 from 'fine-uploader-wrappers/s3'
+import Gallery from 'react-fine-uploader'
+import 'react-fine-uploader/gallery/gallery.css'
+import Results from './Results'
+
+const uploader = new FineUploaderS3({
+    options: {
+        request: {
+            endpoint: process.env.EXPECTED_HOST_NAME,
+            accessKey: process.env.REACT_APP_AWS_ACCESS_KEY_ID
```

Comment:
these things should be handled by webpack config instead of env vars since it's client side